mempool/mempool_test.go

```diff
@@ -135,18 +135,21 @@ func TestRestoreMempool(t *testing.T) {
 	require.Equal(t, 0, pool.Len())
 
 	// push multiple transactions to empty mempool (1,2,3)
+	transactions := make([]mempool.BroadcastedTransaction, 3)
```

Comment:
Also a nitpick, this could be named `expectedTxs` which let the reader know this value is going to be used for test comparison